h11/_abnf.py

```diff
@@ -125,5 +125,5 @@
 chunk_header = (
     r"(?P<chunk_size>{chunk_size})"
     r"(?P<chunk_ext>{chunk_ext})?"
-    r"\r\n".format(**globals())
+    r"\s*\r\n".format(**globals())
```

Comment:
I think this should be `OWS` rather than `\s`, especially as I think the latter allows CRLF

Reply:
Done!